sky/spot/spot_utils.py

```diff
@@ -581,8 +581,13 @@ def is_spot_controller_up(
           identity.
     """
     try:
+        # Set force_refresh=False to make sure the refresh only happens when the
+        # controller is INIT/UP. This optimization avoids unnecessary costly
+        # refresh when the controller is already stopped. This optimization is
+        # based on the assumption that the user will not start the controller
+        # manually from the cloud console.
```

Comment:
Seems safe to me after reading all usage in `core` and `cli`. Even if users manually start the controller in the console, this indicates it was previously autostopped, which means job have finished, so those usage seem safe.